### PR TITLE
SEOImages: User Story 1463880, Part 25

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031900059/Report/

**Comment by damabe** [comment]
Updated based on other PR review feedback. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032600057/Report/

**Comment by mairaw** [comment]
Closed and reopened since one of the images was missing:
image

**Comment by damabe** [comment]
@mairaw fixed the path to the image. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019033000033/Report/

File: docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ You can add functionality to the application model by overriding the `Overridabl
...
   
- ![Visual Basic Application Model &#45;&#45; Run](../../../visual-basic/developing-apps/customizing-extending-my/media/vb_modelrun.gif "VB_ModelRun")  
+ ![Diagram showing the Application Model call sequence.](./media/extending-the-visual-basic-application-model/application-model-call-sequence.gif)  
   
  The Visual Basic Application Model also provides the <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.StartupNextInstance> and <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.UnhandledException> events. The following graphics show the mechanism for raising these events.  
   
- ![Visual Basic Application Model &#45;&#45; Next Instance](../../../visual-basic/developing-apps/customizing-extending-my/media/vb_modelnext.gif "VB_ModelNext")  
+ ![Diagram showing the Application Model next instance.](./media/extending-the-visual-basic-application-model/application-model-next-instance.gif)
```
Comment (mairaw) [inline docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md]:
```suggestion
 ![Diagram showing the OnStartupNextInstance method raising the StartupNextInstance event.](./media/extending-the-visual-basic-application-model/raise-startupnextinstance-event.gif)  
```

File: docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ You can add functionality to the application model by overriding the `Overridabl
...
+ ![Diagram showing the Application Model call sequence.](./media/extending-the-visual-basic-application-model/application-model-call-sequence.gif)  
   
  The Visual Basic Application Model also provides the <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.StartupNextInstance> and <xref:Microsoft.VisualBasic.ApplicationServices.WindowsFormsApplicationBase.UnhandledException> events. The following graphics show the mechanism for raising these events.  
   
- ![Visual Basic Application Model &#45;&#45; Next Instance](../../../visual-basic/developing-apps/customizing-extending-my/media/vb_modelnext.gif "VB_ModelNext")  
+ ![Diagram showing the Application Model next instance.](./media/extending-the-visual-basic-application-model/application-model-next-instance.gif)  
   
- ![Visual Basic Application Model Unhandled Exception](../../../visual-basic/developing-apps/customizing-extending-my/media/vb_unhandex.gif "VB_UnhandEx")  
+ ![Diagram showing the Application Model unhandled exception.](./media/extending-the-visual-basic-application-model/application-model-unhandled-exception.gif)
```
Comment (mairaw) [inline docs/visual-basic/developing-apps/customizing-extending-my/extending-the-visual-basic-application-model.md]:
```suggestion
 ![Diagram showing the OnUnhandledException method raising the UnhandledException event.](./media/extending-the-visual-basic-application-model/raise-unhandledexception-event.gif)  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few suggestions left.

**Comment by damabe** [comment]
@mairaw These changes have been made. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019040200062/Report/

**Review by mairaw — APPROVED** [review]
Thanks!

**Pull request merged**